modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItem.js

```diff
@@ -221,11 +221,17 @@ function CartItem({
 					</div>
 				</div>
 			) : (
-				<a
-					className="mini-cart-item-details"
-					data-senna-off="true"
-					href={productPageUrl}
-				>
+				<div className="mini-cart-item-details position-relative">
```

Comment:
I don't get this: now the if and the else blocks seem to be equal. Is it correct? And in this case why do we need the if?